scripts/engine

```diff
@@ -12,6 +12,17 @@ def main():
     challenge = rospy.get_param("~challenge",
                                 None)  # Challenge to load grammar and command_target from if `grammar` and `command_target` are empty
 
+    # Which grammar target to use for what field of missing information?
+    grammar_target_for_missing_info = rospy.get_param("~grammar_target_for_missing_info", {})
+
+    grammar_target_for_missing_info = {
```

Comment:
Getting a value from the parameter server and then immediately overwriting it with a default(?) looks like the wrong way around. Perhaps swap both assignments, and use the variable as default in the parameter server retrieval call as default value?

Reply:
was already fixed I see. sorry